### PR TITLE
adding imversed testnet assets and pools to osmosis testnet frontend

### PR DESCRIPTION
**Pull request by greggsoft**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+				coinGeckoId: 'imversedtest',
+			},
+		],
+		gasPriceStep: {
+			low: 0.00025,
+			average: 0.0025,
+			high: 0.025,
+		},
+		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
+		explorerUrlToTx: 'https://tex-t.imversed.com/transactions/${txHash}',
```
Comment (JeremyParish69) [inline src/config.ts:1424]: please remove the dollar sign '$'

File: src/config.ts [hunk 2]
```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+	{
+		rpc: 'https://tx-endpoint-test.imversed.com:443',
+		rest: 'https://query-endpoint-test.imversed.com:443',
+		chainId: 'imversed-test-1',
+		chainName: 'imversedtestnet',
+		stakeCurrency: {
+			coinDenom: 'IMV',
+			coinMinimalDenom: 'nimv',
+			coinDecimals: 6,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/imversed.png',
```
Comment (JeremyParish69) [inline src/config.ts:1395]: please upload the imversed tokon logo image(s) to /public/assets/tokens/ and instead title it imv.png, not imversed.png

File: src/config.ts [hunk 3]
```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+		},
+		bip44: { coinType: 118 },
+		bech32Config: Bech32Address.defaultBech32Config('imv'),
+		currencies: [
+			{
+				coinDenom: 'IMV',
+				coinMinimalDenom: 'nimv',
+				coinDecimals: 6,
+				coinImageUrl: window.location.origin + '/public/assets/tokens/imversed.png',
+				coinGeckoId: 'imversedtest',
```
Comment (JeremyParish69) [inline src/config.ts:1406]: is 'imversedtest' really a CoinGecko ID? I couldn't find it in here: https://api.coingecko.com/api/v3/coins/list

File: src/config.ts [hunk 4]
```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+		chainName: 'imversedtestnet',
+		stakeCurrency: {
+			coinDenom: 'IMV',
+			coinMinimalDenom: 'nimv',
+			coinDecimals: 6,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/imversed.png',
+			coinGeckoId: 'imversedtest',
+		},
+		bip44: { coinType: 118 },
+		bech32Config: Bech32Address.defaultBech32Config('imv'),
```
Comment (JeremyParish69) [inline src/config.ts:1399]: Please register the bech32 prefix imv here:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
There are a few things to be addressed. I've left comments where required

File: src/config.ts (hunk 1 above)
Comment (greggsoft) [reply src/config.ts]: Fixed

File: src/config.ts (hunk 2 above)
Comment (greggsoft) [reply src/config.ts]: Image uploaded and name of it changed to imv.png

File: src/config.ts (hunk 3 above)
Comment (greggsoft) [reply src/config.ts]: Can I just comment coinGeckoId param?

File: src/config.ts (hunk 4 above)
Comment (greggsoft) [reply src/config.ts:1399]: I make PR to register `imv` prefix https://github.com/satoshilabs/slips/pull/1306

Comment (greggsoft) [reply src/config.ts:1399]: PR merged and prefix imv registered

Comment (JeremyParish69) [reply src/config.ts:1399]: FYI imv is registered in the Mainnet column, not Testnet. I suppose that be addressed later

**Comment by JeremyParish69** [comment]
LGTM

File: src/config.ts [hunk 5]
```diff
@@ -1376,6 +1382,47 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+		rpc: 'https://tx-endpoint-test.imversed.com:443',
+		rest: 'https://query-endpoint-test.imversed.com:443',
+		chainId: 'imversed-test-1',
+		chainName: 'imversedtestnet',
+		stakeCurrency: {
+			coinDenom: 'IMV',
+			coinMinimalDenom: 'nimv',
+			coinDecimals: 6,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/imv.png',
+			// coinGeckoId: 'imversedtest',
```
Comment (JeremyParish69) [inline src/config.ts:1396]: Can define the coinGeckoId to be 'pool:nimv' ? I see it defined as the alternative coin id
`coinGeckoId: 'pool:nimv'`

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
Pending one last question about the price data/coinGeckoId...

**Pull request merged**